manifests/machineconfigcontroller/events-clusterrole.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: machine-config-controller-events
+  namespace: {{.TargetNamespace}}
+rules:
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["create", "patch"]
```

Comment:
I don't know enough but is patch needed 🤔 is firing an event involved into patch?

Reply:
Probably not, but I just copied this from the MCD ones.

Reply:
Updated to remove `patch` from both (though I didn't test locally).